### ISSUE TITLE
Need a new Logo for EMS

### ISSUE THREAD

**Issue by tharun143** [tharun143/PSoC-EMS#3]
Well, after setting up the application you can see the logo of EMS at the end of the explore page. The existing logo is cool. But I would recommend a new logo for this project which should picture the online accounting software.

I would recommend a logo with either using calligraphy or maybe something like an abstract logo. 
It's up to the creator. 

**Work to be done:**
1. A new logo should be submitted.
2. Upon acceptance of the logo, the existing logo should be replaced with a new one.
3. A favicon image should be added

**Comment by Ayushgup85** [comment]
Can i work on this issue?

**Comment by tharun143** [comment]
Yes @Ayushgup85 please go ahead and join the hitter room for further communication.
